conf/application.conf

```diff
@@ -148,11 +148,7 @@ urls {
   login         = "http://localhost:9949/auth-login-stub/gg-sign-in"
   loginContinue = "http://localhost:9781/trusts-registration"
   logout        = "http://localhost:9514/feedback/trusts"
-  agentSubscriptions = "http://localhost:9437/agent-subscription/start"
   maintainATrustOverview = "http://localhost:9788/maintain-a-trust/overview"
-  maintain {
-    alreadyClaimed = "http://localhost:9788/maintain-a-trust/status/already-claimed"
```

Comment:
Was this not used?

Reply:
No, it wasn't used any more. It was also oddly nested.